src/import.ml

```diff
@@ -1,3 +1,42 @@
 module Int63 = Optint.Int63
 
 type int63 = Int63.t
+
+module Logs = struct
+  let default_reporter ?(prefix = "") () =
+    let report src level ~over k msgf =
+      let k _ =
+        over ();
+        k ()
+      in
+      let ppf = match level with Logs.App -> Fmt.stdout | _ -> Fmt.stderr in
+      let with_stamp h _tags k fmt =
+        let dt = Unix.gettimeofday () in
```

Comment:
Not sure we want to use `gettimeoday` here

Reply:
I've pushed a commit that switches it to `Mtime` instead. Could also use `Ptime`, but I'm not sure which is more applicable here.